frontend/src/app/commonComponents/MultiSelect/MultiSelectDropdown/MultiSelectDropdown.tsx

```diff
@@ -266,7 +267,14 @@ export const MultiSelectDropdown = ({
       !newTarget ||
       (newTarget instanceof Node && !containerRef.current?.contains(newTarget));
 
-    if (newTargetIsOutside && !state.isExtended) {
+    const blurConditionsForRegularList =
+      !state.isExtended && newTargetIsOutside;
+
+    // don't blur if we're clicking within the list but outside the input (ie on table labels)
+    const blurConditionsForExtendedList =
+      state.isExtended && newTargetIsOutside && !state.isOpen;
+
+    if (blurConditionsForRegularList || blurConditionsForExtendedList) {
```

Comment:
have to do this weird workaround because we're bluring the outside click in four separate ways because of the case-wise nature of the regular / extended version of component. 

During pairing Mike and I discussed just refactoring this to be two separate components, which I think we should do because the code right now is a bit of a janga tower. Ticket for that [here](https://github.com/CDCgov/prime-simplereport/issues/6942)